client/components/mma/accountoverview/PersonalisedHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/react';
+import { headline, space } from '@guardian/source-foundations';
+import type { MembersDataApiResponse } from '../../../../shared/productResponse';
+import { isProduct, sortByJoinDate } from '../../../../shared/productResponse';
+
+interface PersonalisedHeaderProps {
+	mdapiResponse: MembersDataApiResponse;
+}
+
+export const PersonalisedHeader = ({
+	mdapiResponse,
+}: PersonalisedHeaderProps) => {
+	const userDetails = mdapiResponse.user;
+
+	const productDetails = mdapiResponse.products
+		.filter(isProduct)
+		.sort(sortByJoinDate);
+	const oldestProduct = productDetails[productDetails.length - 1];
+	const supportStartYear = new Date(oldestProduct.joinDate).getFullYear();
+
+	const calculatedTimeOfDay = () => {
+		const userDate = new Date().getHours();
+
+		if (userDate >= 0 && userDate <= 11) {
```

Comment:
You don't need the first part of these conditions - you can achieve the same thing just checking <= 11 for morning, <= 17 for afternoon and the rest will fall into an else block

Reply:
Yeah that's a nice way to keep it a bit simpler!